machida/cpp/python-wallaroo.c

```diff
@@ -239,12 +239,64 @@ extern PyObject *initial_state(PyObject *computation)
   return pState;
 }
 
+extern PyObject *initial_accumulator(PyObject *aggregation)
+{
+  PyObject *pFunc, *pState;
+
+  pFunc = PyObject_GetAttrString(aggregation, "initial_accumulator");
+  pState = PyObject_CallFunctionObjArgs(pFunc, NULL);
+  Py_DECREF(pFunc);
+
+  return pState;
+}
+
+extern void aggregation_update(PyObject *aggregation, PyObject *data, PyObject *acc)
+{
+  PyObject *pFunc, *pState;
+
+  pFunc = PyObject_GetAttrString(aggregation, "update");
+  PyObject_CallFunctionObjArgs(pFunc, data, acc, NULL);
+  // PyObject_CallFunctionObjArgs(pFunc, aggregation, data, acc, NULL);
+  // PyObject_CallMethod(aggregation, (char*)"update", data, acc, NULL);
+  Py_DECREF(pFunc);
+}
+
+extern PyObject *aggregation_combine(PyObject *aggregation, PyObject *acc1, PyObject *acc2)
+{
+  PyObject *pFunc, *pState;
+  // PyObject *pState;
```

Comment:
Please remove commented out code in final release

Reply:
addressed